.travis.yml

```diff
@@ -16,13 +16,5 @@ install:
   - pip install --no-cache-dir -U -r requirements.txt
 
 script:
-  - ./check_line_length.py
-  - ./runsphinx-build
-  - |
-    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-      # gets commit message of last commit before pull request merge
-      COMMIT_MESSAGE=$(git log $TRAVIS_PULL_REQUEST_SHA --format=%B -n 1)
-      printf "Checking commit message:\n\n"
-      printf "$COMMIT_MESSAGE\n\n"
-      checkcommit --message "$COMMIT_MESSAGE"
-    fi
+  - openwisp-utils-qa-checks --skip-checkmigrations
+  - ./runsphinx-build
```

Comment:
Add endline! :joy:
PR for qa-checks fails qa checks! :joy:

Reply:
Lol 😂 Fixed it